src/checkstyle/checkstyle.xml

```diff
@@ -239,5 +239,21 @@
 		<module name="io.spring.javaformat.checkstyle.check.SpringCatchCheck"/>
 		<module name="io.spring.javaformat.checkstyle.check.SpringJavadocCheck"/>
 		<module name="io.spring.javaformat.checkstyle.check.SpringJUnit5Check"/>
- 	</module>
+		<!--package-info should contain null-safety annotations-->
+		<!--This module will fail to detect multiline annotations-->
+		<module name="RegexpSinglelineJavaCheck">
+			<property name="id" value="javaDocPackageAnnotations"/>
+			<property name="format" value="(@NonNull|@org\.springframework\.lang\.NonNull)"/>
+			<property name="severity" value="warning"/>
+			<property name="minimum" value="1"/>
+			<property  name="maximum" value="5"/>
+			<property name="message" value="package-info.java is missing null-safety annotations  or has null-safe annotations made up of multiple lines. Please check."/>
```

Comment:
```suggestion
			<property name="message" value="package-info.java is missing required null-safety annotations @NonNullApi and @NonNullFields."/>
```

Spring's `@NonNullApi` and `@NonNullFields` annotations should never span multiple lines.